src/docs/user/references/JobPlannerServiceReference.adoc

```diff
@@ -33,14 +34,17 @@ On the left panel, you can easily see the number of associated workflows to each
 
 This page can also be opened when clicking on the "plan" button on the <<_glossary_workflow_studio, *Studio*>> or the <<_glossary_scheduler_web_interface, *Scheduler Portal*>>.
 
-==== Execution Planning visualisation
+=== Execution Planning visualisation and GANTT
 
-On the <<_glossary_job_planner_portal, *Job Planner Portal*>>, Execution Planning page, you can see the recurrence of calendar definitions.
-You can select one or several Calendar Definitions on the left panel and associated workflows will appear on the right panel. 
+On the <<_glossary_job_planner_portal, *Job Planner Portal*>>, Execution Planning page, you can see the recurrence of <<_glossary_calendars_definition,*Calendar Definitions*>> .
+You can select one or several Calendar Definitions on the left panel and associated workflows will appear on the right panel.
+There are 2 ways to visualize planning of the selected Calendar Definitions:
+    - by workflow, Chronologically or on a standard Calendar
```

Comment:
why does Chronologically start with capital character?

Reply:
Because Denis wants it like that.